paramak/reactor.py

```diff
@@ -596,14 +597,21 @@ def export_2d_image(
 
     def export_html(self, filename="reactor.html"):
         """Creates a html graph representation of the points for the Shape
-        objects that make up the reactor. Note, If filename provided doesn't end
-        with .html then it will be appended.
+        objects that make up the reactor. Shapes are colored by their .color
+        property. Shapes are also labelled by their .name. If filename provided
+        doesn't end with .html then .html will be added.
 
         Args:
-            filename (str): the filename to save the html graph
+            filename: the filename used to save the html graph.
+            facet_splines: If True then spline edges will be faceted. Defaults
+                to True.
+            facet_splines: If True then circle edges will be faceted.Defaults
+                to True.
+            tolerance: faceting toleranceto use when faceting cirles and
+                splines. Defaults to 1e-3.
```

Comment:
I don't see these args in the function arguments: is it a copy pasted code from shape.py ?
+ the types are missing

Reply:
Is it ok to have tpyes in the type hinting instead of in the docs

Reply:
I think as long as users don't necessarly have to go to read the docs to know the types it's ok